js-apps/meep-frontend/src/js/containers/exec/network-characteristics-event-pane.js

```diff
@@ -131,14 +131,21 @@ class NetworkCharacteristicsEventPane extends Component {
     super(props);
 
     this.state = {
-      dialogOpen: false,
-      currentElementType: ''
+      dialogOpen: false
+      // currentElementType: ''
     };
   }
 
+  onNetworkCharacPaneClose(e) {
+    e.preventDefault();
+    setElemFieldVal(this.props.element, FIELD_TYPE, '');
```

Comment:
We should not update the selected element state directly here. Update a copy & dispatch instead. Could we clear the selected element instead of just changing its type?

Reply:
Updated to change the element in store